### ISSUE TITLE
Update SDXL size picker to do SD3/Flux

### ISSUE THREAD

**Issue by scruffynerf** [cubiq/ComfyUI_essentials#89]
the Empty Size Picker node makes a dim 4 latent, not a dim 16...

replace with code to offer all options?

example core SD3 is:
`latent = torch.ones([batch_size, 16, height // 8, width // 8], device=self.device) * 0.0609`

which many people are using when they use that latent for Flux, but the correct would be
`latent = torch.ones([batch_size, 16, height // 8, width // 8], device=self.device) * 0.1159`

(see ComfyUI repo: comfy/latent_formats.py)
